exercises/concept/freelancer-rates/test/freelancer_rates_test.exs

```diff
@@ -31,7 +31,9 @@ defmodule FreelancerRatesTest do
 
     @tag task_id: 2
     test "it doesn't round" do
-      assert FreelancerRates.apply_discount(111.11, 13.5) == 96.11015
+      # We round result to 5th decimal place to accept results that vary marginally
+      # because of slightly different order of operations and float imprecision
+      assert Float.round(FreelancerRates.apply_discount(111.11, 13.5), 5) == 96.11015
```

Comment:
I think we should use `assert_in_delta` here.

Reply:
And if we do, maybe we don't need a comment, but I don't really mind either way...